src/GitVersionCore/Helpers/EnumerableExtensions.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+
+namespace GitVersion.Helpers
```

Comment:
Don't we already have an `Extensions` namespace for extension methods?

Reply:
moved to `Extensions`